src/S_Ecobee1.xml

```diff
@@ -15,6 +15,12 @@
       <dataType>string</dataType>
       <sendEventsAttribute>yes</sendEventsAttribute>
     </stateVariable>
+    <stateVariable>
+      <name>FanMinOnTime</name>
+      <dataType>i4</dataType>
+      <defaultValue>0</defaultValue>
+      <sendEventsAttribute>yes</sendEventsAttribute>
+    </stateVariable>
```

Comment:
Is this stateVariable needed?  I think it this section can be removed, but please argue otherwise in case I'm missing something.